### PR TITLE
Debug option: Don't send email or check phone

### PR DESCRIPTION
**Pull request by mfl-code**
I tried to add an option in env `DEBUG_FAKE_SERVICES=true` for easier debugging. Instead of sending email it displays the link in the email, and instead of sending SMS it accept any code.

Note:
* I checked it only with the flag, I don't have email/SMS service to test it otherwise
* package-lock.json changed automatically.

### CODE REVIEW

File: src/webapp/email-service.js [hunk 1]
```diff
@@ -3,12 +3,17 @@ const base64 = require('base-64');
...
 const {
-  MAILGUN_API_KEY, MAILGUN_MESSAGING_DOMAIN, MAILGUN_SENDER, MAILGUN_SERVICE_DOMAIN
+  MAILGUN_API_KEY, MAILGUN_MESSAGING_DOMAIN, MAILGUN_SENDER, MAILGUN_SERVICE_DOMAIN, DEBUG_FAKE_SERVICES
 } = process.env;
 
 const sendEmail = async ({ to, subject, html }) => {
   console.log('Sent email', { to, subject, html });
   // return mg.messages.create(MAILGUN_MESSAGING_DOMAIN, data);
+  if (DEBUG_FAKE_SERVICES=='true' || DEBUG_FAKE_SERVICES=='True') {
+    console.log('DEBUG_FAKE_SERVICES Flag is used. Mot sending email');
```
Comment (ginnyfahs) [inline src/webapp/email-service.js:13]: Nit: "Mot sending email" should be "Not sending email" right?

Might also be worth specifying in the message that the link will be displayed in the console.

File: src/webapp/verification-schemes/phone.js [hunk 2]
```diff
@@ -19,49 +19,57 @@ const PHONE_CHALLENGE_RETRY_PERIOD = 24;
...
 
 module.exports = {
   name: 'phone',
   
   challenge: async member => {
-    let formData = new FormData();
-    formData.append('To', member.phone);
-    formData.append('Channel', 'sms');
+    if (DEBUG_FAKE_SERVICES=='true' || DEBUG_FAKE_SERVICES=='True') {
+      console.log('DEBUG_FAKE_SERVICES Flag is used. Mot chalenging sms to phone #', member.phone);
```
Comment (ginnyfahs) [inline src/webapp/verification-schemes/phone.js:29]: Similar to above - shouldn't this be "Not sending SMS to phone" ?

File: src/webapp/verification-schemes/phone.js [hunk 3]
```diff
@@ -19,49 +19,57 @@ const PHONE_CHALLENGE_RETRY_PERIOD = 24;
...
+      await member.save();
+    }
   },
 
   verify: async (member, smsCode) => {
-    let formData = new FormData();
-    formData.append('To', member.phone);
-    formData.append('Code', smsCode);
+    if (DEBUG_FAKE_SERVICES=='true' || DEBUG_FAKE_SERVICES=='True') {
+      console.log('DEBUG_FAKE_SERVICES Flag is used. Mot verifying sms to phone #', member.phone);
```
Comment (ginnyfahs) [inline src/webapp/verification-schemes/phone.js:53]: And here. Now I'm wondering if there's something I'm missing with the mot...

**Review by ginnyfahs — APPROVED** [review]
Thanks for taking the initiative to be our first contributor! My comments are nits just to make sure future devs know what to expect. After addressing them you're welcome to go ahead and merge.

File: src/webapp/email-service.js (hunk 1 above)
Comment (mfl-code) [reply src/webapp/email-service.js]: Done

File: src/webapp/verification-schemes/phone.js (hunk 2 above)
Comment (mfl-code) [reply src/webapp/verification-schemes/phone.js]: Done

File: src/webapp/verification-schemes/phone.js (hunk 3 above)
Comment (mfl-code) [reply src/webapp/verification-schemes/phone.js]: Done.

**Comment by mfl-code** [comment]
@ginnyfahs I made the requested changes, but don't have the permission to merge.

**Pull request merged**